src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -4272,186 +4272,217 @@ namespace ngraph
+                        args[0], runtime::cpu::mkldnn_utils::get_input_mkldnn_format(node, 0));
+                    auto result_desc = mkldnn_emitter->build_memory_descriptor(
+                        args[0], runtime::cpu::mkldnn_utils::get_input_mkldnn_format(node, 0));
+
+                    size_t softmax_index = mkldnn_emitter->build_softmax_forward(
+                        input_desc, result_desc, softmax_axis);
+
+                    auto& deps = mkldnn_emitter->get_primitive_deps(softmax_index);
+                    writer << "cpu::mkldnn_utils::set_memory_ptr(ctx, " << to_string(deps[0])
+                           << ", " << args[0].get_name() << ");\n";
+                    writer << "cpu::mkldnn_utils::set_memory_ptr(ctx, " << to_string(deps[1])
+                           << ", " << out[0].get_name() << ");\n";
+
+                    writer << "cpu::mkldnn_utils::mkldnn_invoke_primitive(ctx, "
+                           << to_string(softmax_index) << ");\n";
                 }
-                // else cast arg/out to an Nd array
                 else
                 {
-                    std::string shape1toN;
-                    for (size_t d = 1; d < dims; ++d)
+                    writer.block_begin();
+                    const ngraph::op::Softmax* softmax =
+                        static_cast<const ngraph::op::Softmax*>(node);
+                    auto type = out[0].get_type();
+                    auto shape = out[0].get_shape();
+                    auto dims = out[0].get_shape().size();
+                    auto axes = softmax->get_axes();
+
+                    // create arg/out if 1d
+                    if (dims < 1)
                     {
-                        shape1toN += "[";
-                        shape1toN += std::to_string(shape[d]);
-                        shape1toN += "]";
+                        writer << type << "* arg = " << args[0].get_name() << "\n";
+                        writer << type << "* out = " << out[0].get_name() << "\n";
                     }
+                    // else cast arg/out to an Nd array
+                    else
+                    {
+                        std::string shape1toN;
+                        for (size_t d = 1; d < dims; ++d)
+                        {
+                            shape1toN += "[";
+                            shape1toN += std::to_string(shape[d]);
+                            shape1toN += "]";
+                        }
 
-                    writer << type << " (*arg)" << shape1toN << " = (" << type << " (*)"
-                           << shape1toN << ") " << args[0].get_name() << ";\n";
-                    writer << type << " (*out)" << shape1toN << " = (" << type << " (*)"
-                           << shape1toN << ") " << out[0].get_name() << ";\n";
-                }
+                        writer << type << " (*arg)" << shape1toN << " = (" << type << " (*)"
```

Comment:
seems to be formatted with redundant spaces? did we run `apply-code-format.sh`?

Reply:
yes I did. otherwise I believe style check should have been failed.